scss/components/calendar.scss

```diff
@@ -160,12 +153,16 @@ $fd-calendar-table-width: $fd-calendar-table-item-width * 7 !default;
   }
   &__content {
     padding: fd-space(xs) 0;
+    padding-left: 12px;
+    padding-right: 12px;
```

Comment:
These three lines can be rolled up into the single line: `padding: fd-space(xs) 12px;`